### ISSUE TITLE
Where is the OCR? The app looks identical to tachiyomi for me.

### ISSUE THREAD

**Issue by ghost** [Rattlehead15/tachiyomiOCR#9]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.0](https://github.com/tachiyomiorg/tachiyomi/releases/tag/v0.12.0)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.12.1

### Android version

Android 8.0.0

### Device

Galaxy J5 prime

### Steps to reproduce

1. Download the app
2. Open the app
3. Open some manga
4. ...?
5. Don't profit

### Expected behavior

I dunno, this is my first time downloading the app. But I expected to see *something* different than tachiyomi.

### Actual behavior

Exactly like tachiyomi? Do I have to fill all of these?

**Comment by Rattlehead15** [comment]
Wait, did github make you fill all that in? That may be from the original tachiyomi app lol. If you clicked the link to update then it sent you to the original one. You should download tachiyomi ocr from [here](https://github.com/Rattlehead15/tachiyomiOCR/releases/tag/v0.3.1-ocr). If the app tells you to update, please check if it's telling you to update to a tachiyomiOCR version or a new tachiyomi version. If it's the latter then please open up another issue. Sorry for the confusion, really 90% of this app is just copy pasted from tachiyomi and that causes problems sometimes lol

**Comment by ghost** [comment]
But that's the link I originally downloaded it from. Furthermore, I tried downloading an earlier version and this time I found a dictionary lookup button and next to it I found button that allows me to highlight parts of the image. I assume it's there to select the parts you want to OCR. But it does nothing and there is no way to make it go away (you have to close the manga and then open it again).

**Comment by Balt91** [comment]
Hi, I'm having the same issue. I installed the latest version but it seems identical to Tachiyomi (can't find a dictionary or ocr function). In the settings I do find an OCR settings menu, but when I click it to enter, nothing happens. I've so far tried v0.3.1-ocr, both these apk's:

tachiyomiocr-arm64-v8a.apk
61.5 MB
tachiyomiocr.apk
96.7 MB

With and without the original Tachiyomi installed next to it. 
I'll next try to do a fresh install of v0.3.0-ocr and report back. 

_My device is a Lenovo M10 FHD Plus. 
Android 10_

**EDIT:**
I've now tried v0.3.0-ocr as well, no change. 
Next I'll try v0.2-ocr.

**EDIT #2:**
Sadly, v0.2-ocr will not install.

@Rattlehead15 Any ideas?
